database/mockDataGen.js

```diff
@@ -0,0 +1,48 @@
+const faker = require('faker');
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const writer = csvWriter();
+
+// const generateRandomComments = () => {
+//   randomNumber = faker.random.number({
+//     min: 1,
+//     max: 5
+//   });
+//   let comments = [];
+//   for (let i = 0; i < randomNumber; i++) {
+//     comments.push({
+//       username: faker.internet.userName(),
+//       comment: faker.lorem.sentence(),
+//       profileImgUrl: faker.internet.avatar()
+//     });
+//   }
+//   return JSON.stringify(comments);
+// }
+
+
+// let randomComments = generateRandomComments();
+// console.log(randomComments);
+// console.log(randomComments.slice(0, randomComments.length - 2).split(";,"));
+
+const createCSVdata = () => {
+  console.time('timing seed');
+  writer.pipe(fs.createWriteStream('comments.csv'));
+  for (let i = 0; i < 10000000; i++) {
+    const album_id = i;
```

Comment:
Is this a foreign key you are defining here?

Reply:
If you are referring to album_id it's just a naming convention I've been using.

Reply:
sorry let me rephrase! is this meant to be to match up comments to album_ids (which would work as a pseudo foreign key?) just wondering since upon insert the id would be auto generated anyway if its in your schema right? esp since you are just incrementing and not assigning random numbers as album_id

Reply:
Oh ok, I see what you're saying! I'll look into this further. Thanks.